### PR TITLE
Add logic to handle missing transaction index

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana Essilfie-Conduah <nana@swirldslabs.com>

**Description**:
It seems occasionally `transaction_index` can be undefined from mirror node.
Currently the relay throws errors in this case.
While we figure out why

- add logic to manage undefined transaction_index
- add tests

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/lib/eth.ts [hunk 1]
```diff
@@ -1068,7 +1068,7 @@ export class EthImpl implements Eth {
   }
 
   static nullableNumberTo0x(input: number | BigNumber): string | null {
-    return input === null ? null : EthImpl.numberTo0x(input);
+    return input === null || input === undefined ? null : EthImpl.numberTo0x(input);
```
Comment (dimitrovmaksim) [inline packages/relay/src/lib/eth.ts:1071]: The condition can be combined into `input == null`, this way it checks for both null and undefined

**Review by dimitrovmaksim — APPROVED** [review]
LG

**Review by georgi-l95 — DISMISSED** [review]
LG

File: packages/relay/src/lib/eth.ts (hunk 1 above)
Comment (Nana-EC) [reply packages/relay/src/lib/eth.ts:1071]: AH good call, let me make this change and push

Comment (Nana-EC) [reply packages/relay/src/lib/eth.ts]: Updated

**Review by dimitrovmaksim — APPROVED** [review]

**Review by georgi-l95 — APPROVED** [review]

**Pull request merged**